packages/core/src/indexing/tables-manager.ts

```diff
@@ -269,7 +269,7 @@ export class PostgresTablesManager extends TablesManager {
    * @param args
    */
   async hasMidIndices(tableName: string, args: IndexModelArgs): Promise<boolean> {
-    const expectedIndices = indices(tableName).indices.flatMap((index) => index.name)
+    const expectedIndices = defaultIndices(tableName).indices.flatMap((index) => index.name)
     if (args && args.indices) {
```

Comment:
If we're removing the ability to save the args in the database, then we'll want to remove this check as well. As it is, if composedb sets indices for a model, ceramic is shutdown, then brought back up with the model specified in config, we will have a migration failure, since indices are present that shouldn't be.

Reply:
this PR doesn't remove tracking the args in the database.  I'm still exploring if there's a way to actually reconstruct the index definitions just from the native index metadata in the database or not.  If I go that route though that will be in a different PR.  This PR has no functionality change, just cleanup.

> As it is, if composedb sets indices for a model, ceramic is shutdown, then brought back up with the model specified in config

I'm not quite sure what you mean by "brought back up with the model specified in config"?

Reply:
Thought the hooks still existed, but looks like config hooks are gone now. Should be good to go then.